redfish-core/lib/event_service.hpp

```diff
@@ -40,37 +33,303 @@ static constexpr const std::array<const char*, 3> supportedRetryPolicies = {
     "TerminateAfterRetries", "SuspendRetries", "RetryForever"};
 
 #ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
-static constexpr const std::array<const char*, 2> supportedResourceTypes = {
-    "IBMConfigFile", "Task"};
+static constexpr const std::array<const char*, 8> supportedResourceTypes = {
```

Comment:
can we safely ignore this extra resource type?

Reply:
which is extra resource type?
these three resource types added above
static constexpr const std::array<const char*, 3> supportedRegPrefixes = {
    "Base", "OpenBMC", "TaskEvent"};